### ISSUE TITLE
enableMultiCamera always returns 1. It succeeds even if it returns 1.

### ISSUE THREAD

**Issue by 345ml** [AgoraIO-Extensions/react-native-agora#634]
**Describe the bug**
enableMultiCamera always returns 1.
It succeeds even if it returns 1.

**To Reproduce**
This is my code

```
react-native-agora@^4.1.2:
  version "4.1.2"
  resolved "https://registry.yarnpkg.com/react-native-agora/-/react-native-agora-4.1.2.tgz#ef2554cd610dfbf52f9ff8c2363db4fcbd03312d"
  integrity sha512-hdI68BXPYOHT9ndzpBr8d9D6iCrJH8NF5y18bvU4Lqmb7khU0HzyhYj01OBn1tlaL8XiJBtCIhsVPsDvpZoxIQ==
  dependencies:
    buffer "^6.0.3"
    json-bigint "^1.0.0"
    ts-interface-checker "^1.0.2"
```

```js
rtcEngineRef.current = createAgoraRtcEngine();
rtcEngineRef.current.initialize({
  appId: Config.AGORA_APP_ID,
});

const videoEncoderConfiguration = new VideoEncoderConfiguration();
rtcEngineRef.current.setVideoEncoderConfiguration(
  videoEncoderConfiguration,
);
rtcEngineRef.current.setCameraDeviceOrientation(0, 270);

rtcEngineRef.current.addListener(
  'onFirstLocalVideoFrame',
  onFirstLocalVideoFrame,
);

rtcEngineRef.current.setAudioProfile(
  AudioProfileType.AudioProfileSpeechStandard,
  AudioScenarioType.AudioScenarioMeeting,
);

if (Platform.OS === 'ios') {
  createPlugin(rtcEngineRef.current.getNativeHandle());
  enablePlugin();
}

rtcEngineRef.current.enableVideo();

const cameraCapturerConfiguration = new CameraCapturerConfiguration();
cameraCapturerConfiguration.cameraDirection = CameraDirection.CameraRear;
const result = rtcEngineRef.current.enableMultiCamera(true, cameraCapturerConfiguration);
rtcEngineRef.current.startPreview(VideoSourceType.VideoSourceCameraPrimary);
rtcEngineRef.current.startPreview(
  VideoSourceType.VideoSourceCameraSecondary,
);

// This is show 1
alert(result)

rtcEngineRef.current.startSecondaryCameraCapture(
  cameraCapturerConfiguration,
);
```

**Expected behavior**
Return 0.

**Smartphone (please complete the following information):**
 - Device: iPhone SE (A22296) (2nd gen)
 - OS: iOS 16.1.1

**Comment by LichKing-2234** [comment]
I will take a look.

**Comment by LichKing-2234** [comment]
Will be fixed at the next version.

**Comment by 345ml** [comment]
Nice!!!

**Comment by LichKing-2234** [comment]
Will be fixed at 4.2.2

**Comment by 345ml** [comment]
Good job.
Nice DEV!
